Foundation/NSXMLNodeOptions.swift

```diff
@@ -43,55 +43,55 @@
     @constant NSXMLDocumentIncludeContentTypeDeclaration Include a content type declaration for HTML or XHTML
 */
 
-#if arch(x86_64) || arch(arm64)
-public typealias NSXMLNodeOptionRawType = Int
+#if arch(x86_64) || arch(arm64) || arch(powerpc64le)
```

Comment:
so we have to keep bumping this when we get other 64 bit platforms? mips etc? is there not a swift defined or 64 bit?